bcc/cron-manager-bundle/3.1/config/packages/bcc_cron_manager_bundle.yaml

```diff
@@ -0,0 +1,3 @@
+security:
```

Comment:
When installing this recipe on a new installation I've got the error that "security" is not found.

Reply:
I'll check, thanks.

Reply:
Hi @Nyholm, since symfony/security-bundle is not installed by default in Symfony 4, it can be added and there's no an issue with the "security" configuration. I can add it as a dependency in the composer.json file of bcc/cron-manager-bundle, but I'm thinking if this should not be a decision of the user. Instead of adding the security dependency, I'm thinking just to display in the post-install-output configuration the suggestion for the user to add him self the security bundle, if he wants to secure the /admin/cron-manager route. Which approach do you think is better?

Reply:
The idea is that the bundle should work out of the box. If you have a real dependency on symfony/security-bundle you should put that in your composer.json. 

If you want to let the user decide if the security bundle should be used or not, then make sure to change the logic in the bundle so it works (out of the box) without the security bundle.

Reply:
I added symfony/security-bundle in composer.json.